tests/test_lightgbm_converter.py

```diff
@@ -225,8 +225,9 @@ def test_float64_lgbm_perf_tree_trav_regressor_converter(self):
         self._run_float64_lgbm_regressor_converter(1000, extra_config={"tree_implementation": "perf_tree_trav"})
 
     # Random forest in lgbm
+    @unittest.skipIf(True, reason="boosting_type=='rf' produces different probabilites.")
```

Comment:
Will this test ever be used?  Is this due to the runtime error added on lightgbm.py:69-72? If so, can you please add a comment?

Reply:
I let the test because I assume this case will be supported. I added a comment in that way. I can remove the test.

Reply:
a comment is perfect, thanks!